op-node/rollup/derive/deriver.go

```diff
@@ -64,6 +66,18 @@ func (ev PipelineStepEvent) String() string {
 	return "pipeline-step"
 }
 
+// DepositsOnlyPayloadAttributesRequestEvent requests a deposits-only version of the attributes from
+// the pipeline. It is sent by the engine deriver and received by the PipelineDeriver.
+// This event got introduced with Holocene.
+type DepositsOnlyPayloadAttributesRequestEvent struct {
+	ParentHash  common.Hash
```

Comment:
Maybe worth changing it to `eth.BlockID`, so we can follow what block-height this is about, instead of just having to rely on a block-hash in logging.